action.yml

```diff
@@ -118,6 +118,13 @@ inputs:
       Scheduled events default to latest commit of default branch.
       Set this if you want the scheduled job to run on on a specific branch.
     required: false
+  ssh-port:
```

Comment:
I think `git-ssh-port` would be a bit more descriptive, for the input name here and for the variable (`gitSshPort`) in `src/action-ros-ci.ts`